Widget List.ipynb

```diff
@@ -0,0 +1,1495 @@
+{
```

Comment:
Please add any extra notebooks to the `notebooks` directory.

Reply:
This file was supposed to be ignored as is from IPython...

The widgets notebooks are on ihaskell-display/Examples